frontend-react/src/config/index.ts

```diff
@@ -16,6 +16,7 @@ const config = {
     DEFAULT_FEATURE_FLAGS: DEFAULT_FEATURE_FLAGS as string[],
     IS_PREVIEW: envVars.OKTA_URL?.match(/oktapreview.com/) !== null,
     API_ROOT: `${envVars.RS_API_URL}/api`,
+    RS_DOMAIN: "reportstream.cdc.gov",
```

Comment:
Hmmm is this something we can add to the .env.* files as a `REACT_APP_*` variable?  I think it'd be good to keep this dynamic based on the domain the app is currently deployed on.